hilti/src/rt/tests/regexp.cc

```diff
@@ -76,6 +76,49 @@ TEST_CASE("construct") {
                          const regexp::PatternError&);
 }
 
+TEST_CASE("binary data") {
+    // Try binary data.
+    auto input = "\xff\xFF"_b;
+    auto re = RegExp("\\xFF\\xff");
+    CAPTURE(re);
+    CAPTURE(input);
+    CHECK_GT(re.find(input), 0);
+
+    input = "\xf0\xfe\xff"_b;
+    re = RegExp("\xf0\xfe\xff"); // pass in raw data directly
+    CAPTURE(re);
+    CAPTURE(input);
+    CHECK_GT(re.find(input), 0);
+
+    re = RegExp("\\xf0\\xfe\\xff"); // let ctor unescape
+    CAPTURE(re);
+    CAPTURE(input);
+    CHECK_GT(re.find(input), 0);
```

Comment:
Can we just move these to the `find` test? I'd go so far as to remove the manual captures as they we do not expect flakes and adding them back in debugging would be trivial, in which case we could write these as just
```.cpp
CHECK_GT(RegExp("\\xf0\\xfe\\xff").find("\xf0\xfe\xff"_b), 0); // Pass in raw data directly.
CHECK_GT(RegExp("\\xf0\\xfe\\xff").find("\xf0\xfe\xff"_b), 0); // Let ctor unescape.
```

Reply:
Ok, I had just copied that from the ticket.